CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### MIGRATION notes
+
+From this release forward Core will be tested against postgres 11   Existing
```

Comment:
```suggestion
From this release forward Core will be tested against PostgreSQL 11   Existing
```
I know we're not strict about this kind of thing so feel free to ignore this kind of change.